File: AdminPanel/AdminPanel.Web/Modules/ServerTypes/ECommerceWebsite/OrderForm.ts
import { StringEditor, DateEditor, DecimalEditor, IntegerEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface OrderForm {
    Name: StringEditor;
    Address: StringEditor;
    Email: StringEditor;
    Phone: StringEditor;
    UserId: StringEditor;
    DateAdded: DateEditor;
    TotalPrice: DecimalEditor;
    Status: IntegerEditor;
}

export class OrderForm extends PrefixedContext {
    static readonly formKey = 'ECommerceWebsite.Order';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!OrderForm.init)  {
            OrderForm.init = true;

            var w0 = StringEditor;
            var w1 = DateEditor;
            var w2 = DecimalEditor;
            var w3 = IntegerEditor;

            initFormType(OrderForm, [
                'Name', w0,
                'Address', w0,
                'Email', w0,
                'Phone', w0,
                'UserId', w0,
                'DateAdded', w1,
                'TotalPrice', w2,
                'Status', w3
            ]);
        }
    }
}

File: AdminPanel/AdminPanel.Web/Modules/ECommerceWebsite/Order/OrderForm.cs
using Serenity.ComponentModel;
using System;

namespace AdminPanel.ECommerceWebsite.Forms;

[FormScript("ECommerceWebsite.Order")]
[BasedOnRow(typeof(OrderRow), CheckNames = true)]
public class OrderForm
{
    public string Name { get; set; }
    public string Address { get; set; }
    public string Email { get; set; }
    public string Phone { get; set; }
    public string UserId { get; set; }
    public DateTime DateAdded { get; set; }
    public decimal TotalPrice { get; set; }
    public int Status { get; set; }
}